RUN_ME.PY NOT WORKING YET, RUNNING CODE FROM FUNCTIONS.PY
